### PR TITLE
replace std::tr1::unordered_map with std::unordered_map

### PR DESCRIPTION
**Pull request by sduzh**

### CODE REVIEW

File: be/src/runtime/descriptors.h [hunk 1]
```diff
@@ -18,12 +18,12 @@
 #ifndef DORIS_BE_RUNTIME_DESCRIPTORS_H
 #define DORIS_BE_RUNTIME_DESCRIPTORS_H
 
+#include <ostream>
+#include <unordered_map>
 #include <vector>
-#include <tr1/unordered_map>
 #include <vector>
```
Comment (imay) [inline be/src/runtime/descriptors.h:24]: there are two vector

File: be/src/runtime/descriptors.h [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
-#include <boost/scoped_ptr.hpp>
-#include <ostream>
 
+#include <boost/scoped_ptr.hpp>
```
Comment (imay) [inline be/src/runtime/descriptors.h:26]: this can be replaced with std::unique_ptr too

File: be/src/runtime/descriptors.h (hunk 1 above)
Comment (sduzh) [reply be/src/runtime/descriptors.h]: updated

File: be/src/runtime/descriptors.h (hunk 2 above)
Comment (sduzh) [reply be/src/runtime/descriptors.h:25]: It's unused, I removed it.

**Review by imay — APPROVED** [review]
+1 LGTM

**Review by kangkaisen — APPROVED** [review]
+1, LGTM

**Comment by chaoyli** [comment]
It's better to add some description about this pull request.

**Pull request merged**